packages/cli/generators/datasource/templates/datasource.ts.ejs

```diff
@@ -11,4 +12,19 @@ export class <%= className %>DataSource extends juggler.DataSource {
   ) {
     super(dsConfig);
   }
+
+  /**
+   * Start the datasource when application is started
+   */
+  start() {
+    // Add your logic here to be invoked when the application is started
+  }
+
+  /**
+   * Disconnect the datasource when application is stopped. This allows the
+   * application to be shut down gracefully.
+   */
+  stop() {
```

Comment:
When I apply this to by lb4 project, i get:
`The inferred type of 'stop' cannot be named without a reference to '@loopback/repository/node_modules/loopback-datasource-juggler'. This is likely not portable. A type annotation is necessary.`

For me it works only that way:
```typescript
stop(): ValueOrPromise<void> {
```
What I'm doing wrong?

Reply:
I'm adding the return type.